rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -345,7 +355,7 @@ public void before() {
 
         client = ClientBuilder.newClient();
         coordinatorPath = TestPortProvider.generateURL('/' + COORDINATOR_PATH_NAME);
-        server.deploy(Coordinator.class);
+        server.deploy(LRACoordinator.class);
```

Comment:
Please can you explain why is it deploying this test LRACoordinator and not the new `JaxRsApplication`?

Reply:
@xstefank I am sorry but I don't understand why the test would deploy a test `LRACoordinator` Application and not the `JaxRsApplication` which is also an application

Reply:
This is not a JAX-RS container so there is no scanning. I can try to play around it probably by manually adding all classes but this seems easier to me.

Reply:
When you say it's not a JAX-RS container, what are you referring to please? Quarkus, WildFly, both?

Reply:
> This is not a JAX-RS container so there is no scanning. I can try to play around it probably by manually adding all classes but this seems easier to me.

Will it still run in the Undertow container?